### PR TITLE
fix(lsp): remove import-map-remap diagnostic

### LINKED ISSUE #15266: lsp import-map-remap misbehaves with relative map keys
After upgrading to v1.24.0, importing from local files (as long as the module is not in the root directory) triggers below warning:

`The import specifier can be remapped to "./some_dir/bar.ts" which will resolve it via the active import map. deno(import-map-remap)`

Screen Shot 2022-07-21 at 2 16 12 PM

### LINKED ISSUE #17867: [lsp] `deno.jsonc` option to disable `import-map-remap`
Related
https://github.com/denoland/deno/issues/15266

I tried `// deno-lint-ignore import-map-remap`, seems not working.

### PR DESCRIPTION
**Pull request by nayeemrmn**
See https://github.com/denoland/deno/issues/17867#issuecomment-1465286031

Closes #15266.
Closes #17867.

### CODE REVIEW

**Review by dsherret — APPROVED** [review]
This seems reasonable to me, but I never use this functionality so I'll let someone else approve it as well before merging.

**Comment by nayeemrmn** [comment]
Can revisit when #15330 is addressed

**Pull request closed without merging**
